evm/bridge-token-factory/contracts/BridgeTypes.sol

```diff
@@ -2,10 +2,10 @@
 pragma solidity ^0.8.24;
 
 library BridgeTypes {
-    struct FinTransferPayload {
-        uint128 nonce;
-        uint128 origin_nonce;
+    struct TransferMessagePayload {
+        uint128 destination_nonce;
```

Comment:
Should probably use camelCase for solidity

Reply:
Outdated

Reply:
my bad